<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateStreamDialog</class>
 <widget class="QWidget" name="CustomCRSDialog">
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="search_form">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>

        <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout">
            
            <item row="2" >
              <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                  <widget class="QLabel" name="label">
                    <property name="text">
                    <string>Option: </string>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QComboBox" name="modeDropdown">
                  </widget>
                </item>
              </layout>
            </item>

            
            <item row="3" >
              <widget class="QLabel" name="description">
                <property name="text">
                <string> </string>
                </property>
              </widget>
            </item>

            <item row="4" >
              <spacer name="horizontalSpacer">
                <property name="orientation">
                <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
                </property>
              </spacer>
            </item>

            <item row="8">
              <spacer name="horizontalSpacer">
                <property name="orientation">
                <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
                </property>
              </spacer>
            </item>

            <item row="10" >
              <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                  <widget class="QLabel" name="label_survey">
                    <property name="text">
                    <string>Lat, Lon: </string>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QLineEdit" name="surveyPointLat">
                    <property name="placeholderText">
                    <string>lat (y)</string>
                    </property>
                  </widget>
                </item>

                <item>
                  <widget class="QLabel" name="degreeSignY">
                    <property name="text">
                    <string>°</string>
                    </property>
                  </widget>
                </item>

                <item>
                  <widget class="QLineEdit" name="surveyPointLon">
                    <property name="placeholderText">
                    <string>lon (x)</string>
                    </property>
                  </widget>
                </item>

                <item>
                  <widget class="QLabel" name="degreeSignX">
                    <property name="text">
                    <string>°</string>
                    </property>
                  </widget>
                </item>
                

              </layout>
            </item>

            <item row="12" >
              <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                  <widget class="QLabel" name="label_offsets">
                    <property name="text">
                    <string>Lat, Lon: </string>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QLineEdit" name="offsetY">
                    <property name="placeholderText">
                    <string>lat (y)</string>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QLabel" name="offsetYDegreeSign">
                    <property name="text">
                    <string>°</string>
                    </property>
                  </widget>
                </item>

                <item>
                  <widget class="QLineEdit" name="offsetX">
                    <property name="placeholderText">
                    <string>lon (x)</string>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QLabel" name="offsetXDegreeSign">
                    <property name="text">
                    <string>°</string>
                    </property>
                  </widget>
                </item>
                
              </layout>
            </item>

            
            <item row="14" >
              <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                  <widget class="QLabel" name="label_angle">
                    <property name="text">
                    <string>Angle: </string>
                    </property>
                  </widget>
                </item>
                <item>
                  <widget class="QLineEdit" name="rotation">
                    <property name="placeholderText">
                    <string>angle (degrees)</string>
                    </property>
                  </widget>
                </item>

                <item>
                  <widget class="QLabel" name="rotationDegreeSign">
                    <property name="text">
                    <string>°</string>
                    </property>
                  </widget>
                </item>
                
              </layout>
            </item>


            
        </layout>
        </item>


      </layout>
     </item>


    </layout>
   </item>

   
      
    <item row="1" column="0">
      <spacer name="horizontalSpacer">
        <property name="orientation">
        <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
        </property>
      </spacer>
    </item>

    <item row="2" column="0">
      <widget class="QDialogButtonBox" name="dialog_button_box">
      <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Apply</set>
      </property>
      </widget>
    </item>


   </layout>
  </widget>

 
 <resources/>
 <connections/>
</ui>
